Index: models/User.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import mongoose from \"mongoose\";\n\nconst UserSchema = new mongoose.Schema({\n    fullName: {\n        type: String,\n        required: true,\n    },\n    email: {\n        type: String,\n        required: true,\n        unique: true,\n    },\n    passwordHash: {\n        type: String,\n        required: true,\n    },\n    avatarUrl: String,    \n},\n{\n    timestamps: true\n},\n);\n\nexport default mongoose.model('User', UserSchema);
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/models/User.js b/models/User.js
--- a/models/User.js	(revision b106898a08208393119612a71f3932643e566374)
+++ b/models/User.js	(date 1662983001448)
@@ -21,4 +21,6 @@
 },
 );
 
+//rdfffff
+
 export default mongoose.model('User', UserSchema);
\ No newline at end of file
